dockernet/tests/run_all_tests.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+
+# run test files
+INTEGRATION_TEST_FILE=${SCRIPT_DIR}/integration_tests.sh
+
+bash $INTEGRATION_TEST_FILE
```

Comment:
We only have one test for now so should be fine to remove this right?

Reply:
I think we should leave it that way. Although there is only one test now, bui it will be easy to develop if we want to add more tests later.